json_typegen/Cargo.toml

```diff
@@ -17,6 +17,7 @@ travis-ci = { repository = "evestera/json_typegen" }
 
 [dependencies]
 json_typegen_shared = { path = "../json_typegen_shared", version = "0.4" }
+rust_decimal = "1.8.1"
```

Comment:
It should be a default, but I think this should be a feature and a feature-gated dependency.

Reply:
You mean a feature that is gated as part of default?   So..  if `default` is selected then this feature is used?

I would like to try to do this!   I don't think we should require that for this feature to land, I think it should be it's own issue, and feature.

Reply:
Yes, or to rephrase: A feature that is used unless `no-default-features` is selected. 

I agree. It should be its own MR.